Library.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-plusplus */
+export default class Book {
+  constructor(author, Title, Books) {
+    this.author = author;
+    this.Title = Title;
+    this.Books = Books;
+    this.index = 0;
+  }
+
```

Comment:
- [ ] **Optional:** Conventionally, when you modularize your code, every script aside from the `index.js` script is placed inside a folder, named `modules`. If you are to follow this convention, then your `Library.js` file and `luxon.js` should be structured as such.
 
 ```
 modules/Library.js
 modules/luxon.js
```